### PR TITLE
[excel] (Custom XML) Adding section about custom XML parts

### PR DESCRIPTION
**Pull request by AlexJerabek**
This adds a small section on custom XML parts to the Workbook article. It also converts tabs to spaces in the article's code samples.

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[ec1130e](https://github.com/OfficeDev/office-js-docs-pr/pull/619/commits/ec1130e762869425787172807580953b6d4c62d7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CustomXML/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812132252438783-619%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=wZKIKtGL5WarVno5Fkzip2Nd6vFfOOq%2BeDAqtHN8EAc%3D&st=2018-12-13T22%3A48%3A54Z&se=2019-01-13T22%3A53%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/excel-add-ins-workbooks.md [hunk 1]
```diff
@@ -132,17 +132,64 @@ A workbook's settings are similar to the collection of custom properties. The di
...
+        console.log("Workbook needs review : " + needsReview.value);
+    });
 }).catch(errorHandlerFunction);
 ```
 
+## Add custom XML data to the workbook
+
+Excel's Open XML **.xlsx** file format lets your add-in embed custom XML data in the workbook. This data persists with the workbook, independent of the add-in.
+
+A workbook contains a [CustomXmlPartCollection](/javascript/api/excel/excel.customxmlpartcollection), which is a list of [CustomXmlParts](https://docs.microsoft.com/javascript/api/excel/excel.customxmlpart). These give access to the XML strings and a corresponding unique ID. By storing these IDs as settings, your add-in can maintain the keys to its XML parts between sessions.
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/excel-add-ins-workbooks.md]: can also use relative path for CustomXmlParts link

**Comment by o365devx** [comment]
OPS Build status updates of commit _[19c62b6](https://github.com/OfficeDev/office-js-docs-pr/pull/619/commits/19c62b6a656c4e346e157709b8ed706e7cbf20bb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CustomXML/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812132346250577-619%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1bc7HadocS8EYuB%2FTQ9OqVycM3YvWZY0mZptvRPyxDM%3D&st=2018-12-13T23%3A42%3A36Z&se=2019-01-13T23%3A47%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[9dd5b4c](https://github.com/OfficeDev/office-js-docs-pr/pull/619/commits/9dd5b4c874963e3c4ce65c1d7bb38bf06e9dc10f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/excel-add-ins-workbooks.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-CustomXML/docs/excel/excel-add-ins-workbooks.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201812132350064064-619%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=6NjLyehOkG4jSI5SNfHRhrqXkJqVmRIVEiH0KSfoAA8%3D&st=2018-12-13T23%3A46%3A24Z&se=2019-01-13T23%3A51%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/excel-add-ins-workbooks.md [hunk 2]
```diff
@@ -132,17 +132,64 @@ A workbook's settings are similar to the collection of custom properties. The di
...
+        console.log("Workbook needs review : " + needsReview.value);
+    });
 }).catch(errorHandlerFunction);
 ```
 
+## Add custom XML data to the workbook
+
+Excel's Open XML **.xlsx** file format lets your add-in embed custom XML data in the workbook. This data persists with the workbook, independent of the add-in.
+
+A workbook contains a [CustomXmlPartCollection](/javascript/api/excel/excel.customxmlpartcollection), which is a list of [CustomXmlParts](/javascript/api/excel/excel.customxmlpart). These give access to the XML strings and a corresponding unique ID. By storing these IDs as settings, your add-in can maintain the keys to its XML parts between sessions.
```
Comment (ElizabethSamuel-MSFT) [inline docs/excel/excel-add-ins-workbooks.md]: shouldn't the link be to **excel.customxmlparts** instead of excel.customxmlpart?

**Pull request merged**